theseus/geometry/tests/common.py

```diff
@@ -56,3 +56,39 @@ def check_adjoint(group, tangent_vector):
         group_matrix @ group.hat(tangent_vector) @ group.inverse().to_matrix()
     )
     assert torch.allclose(tangent_left.squeeze(2), tangent_right, atol=EPS)
+
+
+def check_projection(Group, Point, Func, batch_size, generator):
```

Comment:
Add a comment reminder that `Func` returns the result of some group operation (e.g., transform_to, inverse, etc.) cost and also populates jacobians in the list passed as the third argument.

Reply:
Updated. Let me know if it is ready to merge.